code/game/objects/items/devices/flash.dm

```diff
@@ -86,8 +86,6 @@
 
 		if(safe <= 0)
 			Subject.Knockdown(10)
-			Subject.flash_eyes(visual = 1, affect_silicon = 1)
```

Comment:
You can see in the OP I git blamed this line and it was the result of some silly replace update thing